src/Analytics.php

```diff
@@ -213,6 +213,7 @@ public function get(
         array $orderBy = [],
         int $offset = 0,
         ?FilterExpression $dimensionFilter = null,
+        ?FilterExpression $metricFilter = null,
```

Comment:
This is currently breaking. Could you move this to that last parameter position so it would be non-breaking?